package.json

```diff
@@ -1,5 +1,12 @@
 {
-    "dependencies": {
-        "yuglify": "^0.1.4"
-    }
+  "scripts": {
+    "test": "eslint dashboard/static/js/*.js; csslint --quiet --warnings=' ' dashboard/static/css/*.css push/static/css/*.css"
```

Comment:
Is there a way to make it run against all `*.js` and `.css` files recursively in the current directory? (Except the `node_modules/` directory.)